eng/scripts/Language-Settings.ps1

```diff
@@ -117,4 +117,62 @@ function Get-python-GithubIoDocIndex() {
   $tocContent = Get-TocMapping -metadata $metadata -artifacts $artifacts
   # Generate yml/md toc files and build site.
   GenerateDocfxTocContent -tocContent $tocContent -lang "Python"
+}
+
+# Updates a python CI configuration json.
+# For "latest", the version attribute is cleared, as default behavior is to pull latest "non-preview".
+# For "preview", we update to >= the target releasing package version.
+function Update-python-CIConfig($pkgs, $ciRepo, $locationInDocRepo, $monikerId=$null){
+  $pkgJsonLoc = (Join-Path -Path $ciRepo -ChildPath $locationInDocRepo)
+
+  if (-not (Test-Path $pkgJsonLoc)) {
+    Write-Error "Unable to locate package json at location $pkgJsonLoc, exiting."
+    exit(1)
+  }
+
+  $allJson  = Get-Content $pkgJsonLoc | ConvertFrom-Json
+  $visibleInCI = @{}
+
+  for ($i=0; $i -lt $allJson.packages.Length; $i++) {
+    $pkgDef = $allJson.packages[$i]
+
+    if ($pkgDef.package_info.name) {
+      $visibleInCI[$pkgDef.package_info.name] = $i
+    }
+  }
+
+  foreach ($releasingPkg in $pkgs) {
+    if ($visibleInCI.ContainsKey($releasingPkg.PackageId)) {
+      $packagesIndex = $visibleInCI[$releasingPkg.PackageId]
+      $existingPackageDef = $allJson.packages[$packagesIndex]
+
+      if ($releasingPkg.IsPrerelease) {
+        if (-not $existingPackageDef.package_info.version) {
+          $existingPackageDef.package_info | Add-Member -NotePropertyName version -NotePropertyValue ""
+        }
+
+        $existingPackageDef.package_info.version = ">=$($releasingPkg.PackageVersion)"
+      }
+      else {
+        if ($def.version) {
+          $def.PSObject.Properties.Remove('version')  
+        }
+      }
+    }
+    else {
+      $newItem = New-Object PSObject -Property @{ 
+          package_info = New-Object PSObject -Property @{ 
+            prefer_source_distribution = "true"
+            install_type = "pypi"
+            name=$releasingPkg.PackageId
+          }
+          excludePath = @("test*","example*","sample*","doc*")
```

Comment:
While you're updating this. Can you change this to `exclude_path` instead of `excludePath`?

Reply:
[PR for Ref](https://github.com/Azure/azure-sdk-tools/pull/1196)